core/src/main/resources/jenkins/model/Jenkins/manage.jelly

```diff
@@ -61,7 +61,7 @@ THE SOFTWARE.
             <j:choose>
               <j:when test="${m.requiresConfirmation}">
                 <l:confirmationLink href="${m.urlName}" post="${m.requiresPOST}" message="${%are.you.sure(m.displayName)}">
-                  <img class="icon" src="${iconUrl}" />
+                  <l:icon class="icon-${icon.replaceAll('.png', '')} icon-xlg" />
```

Comment:
need a better solution, haven't looked at this too much but the API takes the literal file name rather than using classes like l:icon does 😢 

(and UI needs styling too)